config/fixtures/1_glasses.yaml

```
@@ -0,0 +1,70 @@
+sylius_fixtures:
+    suites:
+        cms:
+            listeners:
+                logger: ~
+            fixtures:
+                taxon:
+                    options:
+                        custom:
+                            glasses:
+                                code: "glasses"
+                                name: "Glasses"
+                                slug: "glasses"
+                product:
+                    options:
+                        custom:
+                            Sunshine_Vision_Glasses:
```

It would be nice to use here lowercased snake_case
```suggestion
                            sunshine_vision_glasses:
```